testdata/dnn/onnx/generate_onnx_models.py

```diff
@@ -877,6 +877,51 @@ def forward(self, x, kernel, bias):
 model = ConvBias()
 save_data_and_model_multy_inputs("conv_variable_wb", model, x, kernel, bias)
 
+<<<<<<< HEAD
+class LinearWithConstantInput(nn.Module):
+    def __init__(self, in_dim = 10, const_dim=10, out_dim = 10):
+        super(LinearWithConstantInput, self).__init__()
+        self.in_dim = in_dim
+        self.const_dim = const_dim
+        self.lin_const = nn.Linear(const_dim, out_dim)
+        self.lin_inp = nn.Linear(in_dim, out_dim)
+    def forward(self, x):
+        x = x.reshape(-1, self.in_dim)
+        const = torch.zeros(1, self.const_dim)
+        x_projected = self.lin_inp(x)
+        const_projected = self.lin_const(const)
+        return x_projected*const_projected
+
+x = Variable(torch.rand([1, 5, 10]))
+model = LinearWithConstantInput()
+save_data_and_model("lin_with_constant", x, model)
+
+class MatmulWithTwoInputs(nn.Module):
+    def __init__(self, in_dim = 10, const_dim=10, interm_dim = 10):
+        super(MatmulWithTwoInputs, self).__init__()
+        self.in_dim = in_dim
+        self.const_dim = const_dim
+        self.interm_dim = interm_dim
+        self.linear_for_const = nn.Linear(const_dim, interm_dim)
+        self.first_linear = nn.Linear(in_dim, interm_dim)
+        self.second_linear = nn.Linear(interm_dim, 1)
+    def forward(self, x):
+        x = x.reshape(-1, self.in_dim)
+        x_projected = self.first_linear(x)
+        const = torch.zeros(1, self.interm_dim)
+        const_projected = self.linear_for_const(const)
+        const_projected = const_projected.expand(5, self.interm_dim)
+        sum_tanh = torch.tanh(const_projected + x_projected) 
+        sum_tanh = sum_tanh.reshape(-1, self.interm_dim)
+        sum_tanh_projected = self.second_linear(sum_tanh)
+        sum_tanh_projected = sum_tanh_projected.reshape(1, 5)
+        after_softmax = F.softmax(sum_tanh_projected, dim=1)     
+        return torch.matmul(after_softmax, x)
+
+x = Variable(torch.rand([1, 5, 10]))
```

Comment:
Could you create test data with smaller sizes?

Reply:
Done